components/TileGroup.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react'
+
+export default class TileGroup extends Component {
+  render() {
```

Comment:
**ease of understanding**: why do you switch between functional and class components in your react? I'm not too familiar with the advantages of each, but I feel like you should probably remain consistent about these.

Reply:
Agreed - We should stick to using React functional components rather than class components.